code/game/machinery/portable_turret.dm

```diff
@@ -162,27 +162,25 @@
 var/list/turret_icons

```

Comment:
I /think/ this list originally contained only one thing. And then I removed that one thing. So now it might be useless. I would remove it, but apparently garbage_debug.dm also mentions this list. And I have no clue what that is about. So I think I'd ask for some heads up before removing this.

Reply:
Garbage debug just maintains references to all global lists. Are you compiling with `TESTING`? It's perfectly safe to just remove the `SearchVar()` line.